src/graph/graphcontext.h

```diff
@@ -295,13 +295,36 @@ void GraphContext_DeleteNodeFromIndices
 	Node *n
 );
 
+// delete all references to a node from any relevant index
+void GraphContext_DeleteNodeFromIndicesByLabels
+(
+	GraphContext *gc,
+	Node *n,
+	LabelID *labels,
+	uint label_count
+);
```

Comment:
I would love to consolidate `GraphContext_DeleteNodeFromIndicesByLabels` and `GraphContext_DeleteNodeFromIndices` into a single function:

```c
void GraphContext_DeleteNodeFromIndices
(
	GraphContext *gc,
	Node *n,
	LabelID *labels,  // [optional]
	uint label_count  // [optional] 
);
```